src/lib/internal-icons.js

```diff
@@ -268,6 +268,12 @@ const icons = {
       <path fill="currentColor" fill-rule="evenodd" d="M8 20a3 3 0 0 1-3-3v-5.899a3 3 0 0 1 .953-2.193l4-3.733a3 3 0 0 1 4.094 0l4 3.733A3 3 0 0 1 19 11.101V17a3 3 0 0 1-3 3zm4.682-13.363a1 1 0 0 0-1.364 0l-4 3.733a1 1 0 0 0-.318.731V17a1 1 0 0 0 1 1h1v-3a3 3 0 0 1 2.824-2.995L12 12a3 3 0 0 1 3 3v3h1a1 1 0 0 0 .993-.883L17 17v-5.899a1 1 0 0 0-.318-.73zM12 14a1 1 0 0 0-1 1v3h2v-3a1 1 0 0 0-1-1z"/>
     `,
   },
+  image: {
```

Comment:
I would only suggest changing the name to `asset-pictogram` since it's quite similar to other pictograms we have:
![Captura de Tela 2022-04-01 às 17 40 13](https://user-images.githubusercontent.com/26799272/161338327-39917217-2f25-4ddb-a25a-947922df8364.png)

Reply:
Sorry for the delay! I just pushed the icon renamed. Thanks!